soccer/gameplay/skills/pivot_kick.py

```diff
@@ -66,7 +66,7 @@ def __init__(self):
             'fumble')
 
         # default parameters
-        self.dribbler_power = constants.Robot.Dribbler.MaxPower
+        self.dribbler_power = 60#constants.Robot.Dribbler.MaxPower
```

Comment:
May want to change this so the dribbler value is shared between here and `Capture`.